chart/watermarkpodautoscaler/values.yaml

```
@@ -24,6 +24,10 @@ rbac:
   create: true
 # Configure the controller to watch all namespaces
 watchAllNamespaces: true
+# Labels to be included as metric tags
+# labelsAsTags:
```

we try to have all the fields define.   `{{- if .Values.labelsAsTags }}` still return false if any value is in the list.

```suggestion
labelsAsTags:
```